google/cloud/forseti/services/scanner/dao.py

```diff
@@ -183,6 +188,18 @@ def __repr__(self):
         return string.format(
             self.violation_type, self.resource_type, self.rule_name)
 
+    @staticmethod
+    def update_schema(table):
+        """Maintain all the schema changes for this table.
+
+        Args:
+            table (Table): The table object of this class.
+        """
+        col_name = 'resource_name'
```

Comment:
qq, in the future, as we have more columns to add, how would they be handled?  This time, we want to add `resource_name`, next time we want to add `xyz`. It's hard to see how `xyz` would fit here.

Reply:
You will add the column as an attribute to the class and add the column creation logic in this method, 

i.e.
class Violation(BASE):
      xyz = Column(...)

      def update_schema(table):
             col = Column('resource_name', String(256), default='')
             col.create(table, populate_default=True)

             col = Column('xyz', String(256), default='')
             col.create(table, populate_default=True)

Reply:
Right, that's what I thought.  I think you should make `col_name` into a list of tuples `[(name, type, default), ....]`

And then just loop?

Or else just hardcode the name into the Column(), instead of having a variable `col_name` to track it.

Reply:
Updated to hard code the column name